core/src/main/java/io/questdb/cairo/map/OrderedMapVarSizeRecord.java

```diff
@@ -55,6 +58,7 @@
  * The last accessed key column offset is cached to speed up sequential access.
  */
 final class OrderedMapVarSizeRecord implements OrderedMapRecord {
+    private final BorrowedArray[] arrays;
```

Comment:
there will be an issue with sorting this map on array, single view will be used for both left and right side of the comparison

Reply:
Do we support arrays in ORDER BY?

Reply:
just checked, we went on unbanning streak - i was paranoid - we ban order by